### PR TITLE
add report 'MultilevelCurrentness'

### PR DESCRIPTION
**Pull request by Gigaszi**
### Description
Add new report 'MultilevelCurrentness'.

### Corresponding issue
None

### New or changed dependencies
- None

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [x] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [x] I have commented my code
- [x] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

### CODE REVIEW

File: website/website/index.html [hunk 1]
```diff
@@ -107,6 +107,7 @@ <h3>2. Choose data quality report.</h3>
 						<option value="BuildingReport">Building Report</option>
 						<option value="RoadReport">Road Report</option>
 						<option value="MultilevelMappingSaturation">Multilevel Mapping Saturation</option>
+						<option value="MultilevelCurrentness">Multilevel Mapping Saturation</option>
```
Comment (matthiasschaub) [inline website/website/index.html:110]:
```suggestion
						<option value="MultilevelCurrentness">Multilevel Currentness</option>
```

**Review by matthiasschaub — CHANGES_REQUESTED** [review]

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+MultilevelCurrentness:
+  name: Multilevel Currentness
+  description: |
+    This report shows the currentness of four major Map Features (https://wiki.openstreetmap.org/wiki/Map_features):
+    buildings, land-use/land-cover, points of interest and infrastructure.
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml:6]:
```suggestion
    This report shows the currentness of four major map features (https://wiki.openstreetmap.org/wiki/Map_features):
    buildings, land-use and land-cover, points of interest and infrastructure.
```

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  label_description:
+    red: |
+      The general currentness for central elements in this region is low indicating a very early stage of the data.
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml:9]:
```suggestion
      The general currentness for major map features in this region is low indicating a very early stage of the data.
```

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    yellow: |
+      The currentness of the elements in this region is mediocre. The results for each indicator should be reviewed
+      to determine the suitability of this region.
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml:12]:
```suggestion
      The currentness of major map features in this region is mediocre. The results for each indicator should be reviewed
      to determine the suitability of this region.
```

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    green: |
+      Most of the central elements were edited in recent years. This indicates a high currentness in this region.
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml:14]:
```suggestion
      Most of the major map features were edited in recent years. This indicates a high currentness in this region.
```

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml (hunk 3 above)
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml]: Done

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml (hunk 4 above)
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml]: Done

File: workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml (hunk 5 above)
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml]: Done

**Review by matthiasschaub — CHANGES_REQUESTED** [review]
Good addition to our new set of reports! The changelog needs to be updated, since a new release has been made. Once done this PR can be merged.

**Comment by Gigaszi** [comment]
Done

**Review by matthiasschaub — APPROVED** [review]

**Pull request merged**